our_site/planner/templates/planner/dashboard.html

```diff
@@ -52,6 +52,7 @@
         <div class="box">
             <b>{{ event.name }}</b>
             <p>{{ event.description }}</p>
+            <!-- <p>{{ event.date }}</p> -->
```

Comment:
Why is the event.date commented out?

Reply:
i added the date because it wasn't there but i didn't know whether we wanted to date or not so i commented it out